x-pack/elastic-agent/pkg/agent/install/paths_darwin.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build darwin
+
+package install
+
+const (
+	// BinaryName is the name of the installed binary.
+	BinaryName = "elastic-agent"
+
+	// InstallPath is the installation path using for install command.
+	InstallPath = "/Library/Elastic/Agent"
```

Comment:
I somehow stumbled over this path as I expected it under `/opt` but I'm probably just used to this from homebrew.

What is the reason it is `Elastic/Agent` and not `elastic-agent` or similar in one word? Same question applies to all other OS? Is uppercase recommended? Do we expect other elastic tools to be installed here?

Reply:
Hm, I now stumbled over the elastic-agent shell wrapper below, things are making now more sense.

Reply:
This is so Elastic Endpoint will be next to each other.

Endpoint is installed at `/Library/Elastic/Endpoint` so we install agent at `/Library/Elastic/Agent`. This is the same for all OS's so we can be next to Endpoint, which make it cleaner from a user perspective of seeing everything that is installed from Elastic.

Reply:
Makes sense to align it with Endpoint.